handlers/handlers.go

```diff
@@ -75,6 +75,25 @@ func HandleUserRequest(
 	}
 
 	switch request.Method {
+	case http.MethodGet:
```

Comment:
Why do we need to get all the usernames?

Reply:
To be clear we're getting the users' names, not the usernames (we don't currently have "usernames"). But we're getting them so that we can display the name of the user who made the comment

Reply:
oh if you're asking why do we get the names of ALL of our users, that was just quicker. I was imagining a more refined url with query params, but it didn't seem necessary at this point. And we should talk about what that would look like.